### DISCUSSION TITLE
Bling bling

### DISCUSSION THREAD

**Discussion by piotrwicijowski** [fromITzerotoIThero/personal-website#8] [Ideas]
@fromITzerotoIThero Recalling our recent discussion I was looking for a solution for "how to animate elements once they appear in viewport", and I found an article that should work. Unfortunately it's not a pure css solution, and it looks a bit convoluted, but if you get stuck, just let me know.

 
https://egghead.io/blog/how-to-animate-elements-when-in-view-on-scroll-with-framer-motion

**Comment by fromITzerotoIThero** [comment]
I thank you, thank you! I will check it out! :)))
